src/VirtoCommerce.WebHooksModule.Data/Services/RetriableWebHookSender.cs

```diff
@@ -186,13 +164,13 @@ protected virtual async Task HandleErrorAsync(WebhookWorkItem webHookWorkItem, W
         /// <param name="webHookWorkItem"></param>
         /// <param name="webHookSendResponse"></param>
         /// <returns></returns>
-        protected virtual WebhookFeedEntry GetOrCreateFeedEntry(WebhookWorkItem webHookWorkItem, WebhookSendResponse webHookSendResponse, string errorMessage)
+        protected virtual WebhookFeedEntry GetOrCreateFeedEntry(WebhookWorkItem webHookWorkItem, WebhookSendResponse webHookSendResponse)
         {
             var feedEntry = webHookWorkItem.FeedEntry;
 
             if (feedEntry == null)
             {
-                feedEntry = WebHookFeedUtils.CreateErrorEntry(webHookWorkItem, webHookSendResponse, errorMessage);
+                feedEntry = WebHookFeedUtils.CreateErrorEntry(webHookWorkItem, webHookSendResponse);
```

Comment:
We should not save all errors to the feed, as it could uncontrollably increase database size. Need to limit errors count in feed by some setting - e.g. last 5 errors saved (configurable).
And for feed saving, we should not use our generic `SaveChanges` implementation, as it is a resource expensive. Please add a method allowing to increment attempts count of  a feed entry without trying to load the existing record from the database (possible ways could be found here: https://stackoverflow.com/questions/4218566/update-a-record-without-first-querying)

Reply:
added the logic and use `ExecuteSqlRaw`